README.md

```diff
@@ -37,7 +37,7 @@ CUPyDO currently features interfaces for the following solvers:
   - VLM **[v2.0]** (https://github.com/ulgltas/VLM)
   - A Vortex Lattice Method, developed at the University of Liège.
   ---
-  - PFEM3D **[2.1.0]** (https://github.com/ImperatorS79/PFEM3D)
+  - PFEM3D **[master]** (https://github.com/ImperatorS79/PFEM3D)
```

Comment:
could you make a tag in PFEM3D? "master" will mean nothing in a few weeks/months.
The idea is to be able to rebuild this version of CUPyDO in the future by having a look at this list of versions.

Reply:
Yes that was the plan, I will ask Simon to create a tag of the current version.

Reply:
il semble venir de loin, ce tag

Reply:
Nous ne faisons en effet pas souvent de tag dans PFEM3D, j'avais laissé "Master" pour ne pas oublier de créer un tag juste avant de merge dans CUPyDO. Je dois re-vérifier que tout est à jours avec PFEM3D puisque le Master continue à évoluer rapidement. Je fais ça dés que j'ai terminé la mise à jour (au propre) relative à PaStiX.